doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -129,6 +129,13 @@ Note what happens when you try to move an end-effector out of its reachable work
 .. image:: rviz_plugin_invalid.png
    :width: 700px
 
+Moving Joints or in Null Space
+++++++++++++++++++++++++++++++++++
+You can use the **Joints** tab to control single joints and to move the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in the animation below.
+
+.. image:: rviz_joints_nullspace.webm
```

Comment:
[In other cases](https://ros-planning.github.io/moveit_tutorials/doc/perception_pipeline/perception_pipeline_tutorial.html) we went for animated gifs instead.
That also resolves the question of how to integrate it :grin:

Reply:
I didn't manage to test this by running `./build-locally.sh` because `rosdoc_lite` and `xdg-open` cannot be found, and installing `ros-melodic-rosdoc-lite` didn't solve the problem. I don't remember having issues with this. Is this new? Shouldn't it run in the source container?

If it doesn't work I'll change it to a gif.